terra_notebook_utils/cli/drs.py

```diff
@@ -76,11 +77,18 @@ def drs_head(args: argparse.Namespace):
         setattr(args, 'buffer', MULTIPART_THRESHOLD)
     if not getattr(args, 'bytes', None):
         setattr(args, 'bytes', 1)
```

Comment:
Not part of this PR, but the get/set attr should be removed. The arguments are populated as expected.